jupyterhub/jupyterhub-idle-culler/base/deploymentconfig.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: jupyterhub-idle-culler
+spec:
+  selector:
+    app: jupyterhub-idle-culler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: jupyterhub-idle-culler
+    spec:
+      containers:
+        - name: jupyterhub-idle-culler
+          image: jupyterhub-idle-culler:latest
+          env:
+            - name: JUPYTERHUB_API_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: jupyterhub-api-token
+                  key: token
+          ports:
+            - containerPort: 8080
```

Comment:
I don't think we need to expose a port since it does not have any API endpoint

OR is it for metrics?

Reply:
I can remove it now, and we can add it later if we need it.